diff -uNr gnome-online-accounts-3.6.3/src/goabackend/goautils.c gnome-online-accounts-3.6.3.new/src/goabackend/goautils.c
--- gnome-online-accounts-3.6.3/src/goabackend/goautils.c	2013-03-04 14:17:27.000000000 +0100
+++ gnome-online-accounts-3.6.3.new/src/goabackend/goautils.c	2013-03-11 15:32:31.980162936 +0100
@@ -480,5 +480,5 @@
       break;
     }
 
-  g_set_error (err, GOA_ERROR, GOA_ERROR_SSL, error_msg);
+  g_set_error (err, GOA_ERROR, GOA_ERROR_SSL, "%s", error_msg);
 }
